extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -78,7 +78,7 @@ function axWFactoryTagCheck() {
 	axWFactoryValidateRequest( $wgRequest, $wgUser, __METHOD__ );
 
 	if ( !$wgUser->isAllowed( 'wikifactory' ) ) {
-		return;
+		return '';
 	}
 
 	$tagName = $wgRequest->getVal( "tagName" );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "tagName" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)